src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
OK, there are different detection scenarios.
When you have some dir with files and you
know this is DOS but you try to detect what
kind of DOS is it, this is the DOS type detection.
When you just do "test -d" this is the detection
of existence. I just want to avoid the type detection,
but existence detection is of course unavoidable.
I guess you initially wanted to do both things, which
is too much. With my suggestion the type detection
is removed, so I meant exactly type detection when
saying the detection is not needed.